website/docs/language/resources/ephemeral.mdx

```diff
@@ -0,0 +1,137 @@
+---
+page_title: ephemeral resources configuration reference
+description: Learn about ephemeral resource blocks that you can specify in Terraform configurations. Ephemeral resource blocks represent temporary resources that Terraform does not store in state.
+---
+
+# Ephemeral resource configuration reference
+
+This topic provides reference information for the `moved` block. 
+
+## Introduction
+
+_Ephemeral Resources_ are Terraform resources that are temporary, not stored in
+state, and have a unique lifecycle. Each ephemeral block describes one or more
+ephemeral resources, such as a temporary password or connection to another
+system.
+
```

Comment:
Can we add a list somewhere for where you can use ephemeral resources? Something like this:

```
You can only reference ephemeral resources in specific ephemeral contexts or Terraform throws an error. The following are valid contexts for referencing ephemeral resources:
* In another ephemeral resource
* In [local values](/terraform/language/values/locals#ephemeral-values)
* In [ephemeral variables](terraform/language/values/variables#exclude-values-from-state)
* In [ephemeral outputs](/terraform/language/values/outputs#ephemeral-avoid-storing-values-in-state-or-plan-files)
```

Reply:
Check where I added it, not sure where else?